packages/manager/apps/dedicated/package.json

```
@@ -25,6 +25,7 @@
     "@bower_components/angular-ui-utils": "angular-ui/ui-utils#~0.2.3",
     "@ovh-ux/manager-banner": "^1.1.3",
     "@ovh-ux/manager-beta-preference": "^1.0.0",
+    "@ovh-ux/manager-catalog-price": "^0.0.0 || ^1.0.0",
```

If component has been released we can  
```suggestion
    "@ovh-ux/manager-catalog-price": "^1.0.0",
```

---

The version has been updated